plutus-tx-plugin/test/AsData/Budget/9.6/destructSum-manual.pir.golden

```diff
@@ -89,24 +89,21 @@ in
                           (y : integer)
                           (z : integer)
                           (w : integer) ->
-                           let
-                             !int1_ : integer = addInteger x x
```

Comment:
Hmm, don't we know that this is pure anyway? I thought we knew that e.g. `addInteger` was pure?